src/pages/TrafficList/TrafficListComponent.tsx

```diff
@@ -0,0 +1,335 @@
+import * as React from 'react';
+import {
+  Badge,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Title,
+  Tooltip,
+  TooltipPosition
+} from '@patternfly/react-core';
+import { style } from 'typestyle';
+import {
+  cellWidth,
+  IRow,
+  ISortBy,
+  sortable,
+  SortByDirection,
+  Table,
+  TableBody,
+  TableHeader
+} from '@patternfly/react-table';
+import { Link } from 'react-router-dom';
+import { TrafficItem, TrafficNode } from './TrafficDetails';
+import * as FilterComponent from '../../components/FilterList/FilterComponent';
+import { ThresholdStatus, NA } from 'types/Health';
+import { NodeType, hasProtocolTraffic, ProtocolTraffic } from 'types/Graph';
+import { getTrafficHealth } from 'types/ErrorRate';
+import history, { URLParam } from 'app/History';
+import { createIcon } from 'components/Health/Helper';
+
+export interface TrafficListItem {
+  healthStatus: ThresholdStatus;
+  icon: string;
+  node: TrafficNode;
+  protocol: string;
+  trafficRate: string;
+  trafficPercent: string; // percent success
+}
+
+type TrafficListComponentProps = FilterComponent.Props<TrafficListItem> & {
+  direction: 'inbound' | 'outbound';
+  trafficItems: TrafficItem[];
+};
+
+type TrafficListComponentState = FilterComponent.State<TrafficListItem> & {
+  sortBy: ISortBy;
+};
+
+const columns = [
+  {
+    title: 'Name',
+    transforms: [sortable]
+  },
+  {
+    title: 'Protocol',
+    transforms: [sortable]
+  },
+  {
+    title: 'Rate',
+    transforms: [sortable]
+  },
+  {
+    title: 'Percent',
+    transforms: [sortable, cellWidth(5) as any]
+  },
+  {
+    title: 'TrafficStatus',
```

Comment:
In sublist with status (WorkloadPods, IstioConfigSubList) status is preffered to be the first column.

Reply:
Perhaps I should read all comments first :)  Ignore the question above...